index.php

```diff
@@ -3,11 +3,19 @@
 require_once 'src/connectToDb.php';
 require_once 'src/Models/BlogModel.php';
 
+session_start();
 $db = connectToDb();
-
 $blogModel = new BlogModel($db);
 
 $blogs = $blogModel->getAllPosts();
+
+if ($_SESSION['userid'] === false){
+    $isUserLoggedIn = '<a href="login.php">Login</a>';
+}
+else {
+    $isUserLoggedIn = '';
```

Comment:
See my comment about this on login.php.

$isUserLoggedIn should be either true or false, with an if statement in the HTML that determines whether or not to diplay the login link